internal/app/machined/pkg/system/runner/process/process_test.go

```diff
@@ -224,6 +224,7 @@ func TestProcessSuite(t *testing.T) {
 		t.Skip("wrapperd not found")
 	}
 
+	// What's the purpose of this test? Should it be replaced for new subprocess start method?
```

Comment:
Needs review

Reply:
process reaper is to the fact that Talos machine is PID=1 and it should reap all zombies, so it runs a `waitpid()` loop which breaks Go's wait for the process (as it reaps before Go might have a chance), so unrelated to wrapperd